python/perspective/setup.py

```diff
@@ -66,7 +66,7 @@
 
 requires_aiohttp = ["aiohttp>=3,<4"]
 
-requires_jupyter = ["jupyterlab>=3.2,<4"]
+requires_jupyter = ["jupyterlab>=3.2,<4", "Jinja2>=2.0,<4"]
```

Comment:
IMO this is just a regular requirement. `requires_jupyterlab` is used to install Jupyter Lab in CI, real users will likely already have this installed in their environment and would never think to install the undocumented `pip install perspective-python[jupyterlab]`, making this extra `Jinja` dep easy to miss.

Also please pin this in the CI `requirements` files.

Reply:
I misunderstood what requires_jupyter was for… appreciate the explanation!

Reply:
oops, need to pin this in the requirements file.

Reply:
I bumped the deps using the new update_python script

It made a really noisy diff which was made a lot cleaner once I sorted the lines in the updated requirements-*.txt files. Maybe this is something we could add to the update_python script? (or to some documentation about how to update deps, which I'm happy to write)

Reply:
Added sorting to the requirements.txt updater